src/test/java/com/mongodb/jdbc/MongoDatabaseMetaDataTest.java

```diff
@@ -383,15 +383,25 @@ void testGetSQLKeywords() throws SQLException {
 
     @Test
     void testGetNumericFunctions() throws SQLException {
-        final String expectedFunctions = "";
+        final String expectedFunctions =
+                "ABS,ABS,ABS,ABS,"
```

Comment:
A few of the newly added functions will be added to this list after updating the function category in `MongoFunctions.java`